specification/synapse/resource-manager/Microsoft.Synapse/stable/2021-06-01/bigDataPool.json

```
@@ -584,6 +590,7 @@
           "description": "The maximum number of executors alloted",
           "x-ms-mutability": [
             "create",
+            "read",
             "update"
           ]
```

Rather than adding "read" you should just remove x-ms-mutability altogether,
```suggestion
          "description": "The maximum number of executors alloted"
```